tests/unit/test_main.py

```diff
@@ -45,6 +47,46 @@ def test_get_usage(self, stdout):
         self.assertEqual(exitcode, 0)
         self.assertIn('usage:', output)
 
+    def test_get_erronous_cfg_defaults(self):
+        conf = tempfile.mkstemp()
+        handle = conf[0]
+        filepath = conf[1]
+        self.addCleanup(os.unlink, filepath)
+
+        with os.fdopen(handle, 'w') as conf_file:
+            conf_file.write('aaaaaaaaaaa [Defaults]\n')
+
+        try:
```

Comment:
Could you rewrite this with self.assertRaise() instead ? I find it much more readable:
with self.assertRaises(SystemExit):
   main.get_defaults()